### ISSUE TITLE
Missing option for screen reader for changing present status

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5572]
**Current behavior**
There is almost no feedback if you are changing your own present status and you do not know how you chagned it 

**Wanted behavior**
There should be meaningful feedback if you change your own present status
